delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/cache/KeyValueStore.scala

```diff
@@ -242,7 +243,7 @@ object KeyValueStore {
           // $COVERAGE-ON$
         }
         .retryingOnSomeErrors(retryWhen)
-        .hideErrors
+        .logAndDiscardErrors(s"putting the key '$key' in the Distributed Data '$id'")
```

Comment:
On my future PR, the `retryingOnSomeErrors` won't be silent anymore

Reply:
Same, I've added it anyway as a safeguard.